bridgetown-core/lib/bridgetown-core/rack/routes.rb

```diff
@@ -2,6 +2,12 @@
 
 module Bridgetown
   module Rack
+    @interrupted = false
+
+    class << self
+      attr_accessor :interrupted
```

Comment:
Can we use ActiveSupport's `mattr_accessor` here by any chance?

Reply:
Good question…I think `mattr_accessor` is particularly useful when object instances which have included a module want to access that module's accessor easily. No need for that here, so probably wouldn't need the overhead